Header header
time   init
time   end
string command
string response
